examples/Servo-Stop/Servo-Stop.ino

```diff
@@ -0,0 +1,36 @@
+#include <Servo.h>
+#define trigPin  4
+#define echoPin  2
+
+Servo myservo;  // create servo object to control a servo
+int pos = 0; // variable to store the servo position
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(9600);
+  pinMode(trigPin, OUTPUT);
+  pinMode(echoPin, INPUT);
+  myservo.attach(9);  // attaches the servo on pin 9 to the servo object
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+  long duration, distance;
```

Comment:
```suggestion
  unsigned long duration;
  unsigned int distance;
```
`pulseIn()` returns `unsigned long`.
Maximum range of the common ultrasonic sensor is 400 cm, so an `unsigned int` will be plenty sufficient.

Reply:
PR Updated